backend/grant/proposal/models.py

```diff
@@ -238,10 +271,19 @@ def validate_publishable(self):
     @staticmethod
     def create(**kwargs):
         Proposal.validate(kwargs)
-        return Proposal(
+        proposal = Proposal(
             **kwargs
         )
 
+        # arbiter needs proposal.id
+        db.session.add(proposal)
+        db.session.commit()
```

Comment:
`.flush` instead of `.commit` is more appropriate here - helps us get closer to a transactional API

Reply:
Fixed.